config/default.yml

```diff
@@ -2699,6 +2699,11 @@ Style/DoubleNegation:
   StyleGuide: '#no-bang-bang'
   Enabled: true
   VersionAdded: '0.19'
+  VersionChanged: '1.0'
+  EnforcedStyle: context
+  SupportedStyles:
+    - context
```

Comment:
I think better names would be `forbidden` and `allowed_in_returns` or something along those lines.

Reply:
Or `allowed_in_boolean_context`?

Reply:
Thank you for suggesting a good name! I think `allowed_in_returns ` is clear and straightforward!